legacy/firmware/fsm_msg_ethereum.h

```diff
@@ -19,9 +19,8 @@
 
 static bool fsm_ethereumCheckPath(uint32_t address_n_count,
                                   const uint32_t *address_n, bool pubkey_export,
-                                  uint64_t chain_id) {
-  if (ethereum_path_check(address_n_count, address_n, pubkey_export,
-                          chain_id)) {
+                                  const EthereumDefinitionsDecoded *defs) {
```

Comment:
let's return false when `defs` is null

Reply:
9556b8f18d0dab6fc84e20f31bad8ba707737562